scripts/test-zap.sh

```diff
@@ -13,7 +15,7 @@ while [ $COUNTER -lt $COUNT ]; do
   TARGET=$(cat scripts/targets.json | jq ".targets[${COUNTER}] .url" -r)
 
   echo Scanning $NAME: $TARGET
-  zap-cli quick-scan --spider --scanners all $TARGET
+  zap-cli -v quick-scan --spider --ajax-spider --scanners all $TARGET
```

Comment:
It's a bit slower than not including it, but I haven't seen any significant changes.